modules/serverless-enable-scale-to-zero.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * serverless/admin_guide/serverless-admin-autoscaling.adoc
+
+[id="serverless-enable-scale-to-zero_{context}"]
+= Enabling scale-to-zero
+
+Cluster administrators can enable or disable scale-to-zero for the cluster.
```

Comment:
I would phrase it "...can enable or disable scale-to-zero globally for their Serverless installation". There is cluster autoscaler so it might be confusing as it is now.